gradle/release.gradle

```diff
@@ -41,17 +41,19 @@ gradle.taskGraph.whenReady( { graph ->
 })
 
 gradle.taskGraph.whenReady { taskGraph ->
-  if (!version.endsWith('SNAPSHOT')) {
-    if(!project.hasProperty('signing.keyId') || !project.hasProperty('signing.secretKeyRingFile')) {
-      println "You must configure your signing.keyId and signing.secretKeyRingFile"
-      println "in ~/.gradle/gradle.properties in order to sign jars\n"
-      println "See https://cwiki.apache.org/confluence/display/GEODE/Release+Steps"
-      throw new GradleException("Signing key/keyring is missing")
-    }
+  if (project.hasProperty('askpass')) {
```

Comment:
Should this be called `signReleaseArtifacts` since it guards signing (which may cause the password dialog to appear depending on your environment)?